adala/runtimes/_litellm.py

```diff
@@ -185,6 +193,9 @@ def record_to_record(
             logger.debug(tb)
             return dct
         except Exception as e:
+            # Catch case where the model does not return a properly formatted output
+            if type(e).__name__ == 'ValidationError' and 'Invalid JSON' in str(e):
+                e = ConstrainedGenerationError()
```

Comment:
not sure if this works syntactically (i.e. if you can't do this without `raise`, but would be nice to maintain the lineage here and do `ConstrainedGenerationError() from e`

Reply:
Whats the benefit though? Doesnt give us any additional information, I dont want the type/message of the original exception